src/components/ContentHighlights/HighlightStepper/HighlightStepperFooterHelpLink.jsx

```diff
@@ -2,17 +2,54 @@ import React from 'react';
 import {
   Hyperlink,
 } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import { useContextSelector } from 'use-context-selector';
+import { connect } from 'react-redux';
+import { ContentHighlightsContext } from '../ContentHighlightsContext';
+import EVENT_NAMES from '../../../eventTracking';
+import { STEPPER_HELP_CENTER_FOOTER_BUTTON_TEXT } from '../data/constants';
 
-const HighlightStepperFooterHelpLink = () => (
-  <div>
-    <Hyperlink
-      target="_blank"
-      destination={process.env.ENTERPRISE_SUPPORT_PROGRAM_OPTIMIZATION_URL}
-      className="small"
-    >
-      Help Center: Program Optimization
-    </Hyperlink>
-  </div>
-);
+const HighlightStepperFooterHelpLink = ({ enterpriseId }) => {
+  const stepperModal = useContextSelector(
+    ContentHighlightsContext,
+    v => v[0].stepperModal,
+  );
+  const trackEvent = () => {
+    const trackInfo = {
+      stepper_modal: {
+        highlight_title: stepperModal.highlightTitle,
+        current_selected_row_ids: stepperModal.currentSelectedRowIds,
+        current_selected_row_ids_length: Object.keys(stepperModal.currentSelectedRowIds).length,
+        is_stepper_modal_open: stepperModal.isOpen,
+      },
```

Comment:
I don't think we necessarily need these properties for this particular event. This event solely gets at the question of, "are admins ever clicking the help link?" and these properties don't really give any further insights for that particular question (e.g., knowing the highlight title at the time of click isn't really relevant IMHO).